include/spinnaker.h

```diff
@@ -1100,8 +1100,17 @@ enum {
 
 // On-chip SDRAM
 //! Bits in #GPIO_PORT, also on-chip SDRAM
```

Comment:
Move this line to be immediately above the `enum` declaration, not with intervening declarations.

Reply:
Done